LICENSE

```diff
@@ -199,287 +199,3 @@ Apache License
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
```

Comment:
Hi Moaz,

Please refer to this example created by Apache PMC, https://github.com/justinmclean/ApacheWombat/.
We did bundle some code from other projects, hence we need to include their licenses. We can do in two ways,
1. Add the dependent licenses in the LICENSE file. This is what we are following now.
2. Add the license file of each dependent lib in a separate file and add a link/pointer to that file in LICENSE. Just follow the above example.